apps/system/fxa/test/marionette/lib/fxa.js

```diff
@@ -63,8 +62,8 @@ FxA.Selectors = {
     pwSetInput: '#fxa-pw-set-input',
     pwRefresh: '#fxa-pw-input-refresh',
     COPPAElementId: '#fxa-coppa',
-    COPPASelectId: 'fxa-age-select',
-    COPPAOptionVal: '1990 or earlier',
+    COPPASelectId: '#fxa-age-select',
```

Comment:
"Children's Online Privacy Protection Act"; https://en.wikipedia.org/wiki/Children%27s_Online_Privacy_Protection_Act

Basically we need to do an age check to make sure we're doing the right thing.